### DISCUSSION TITLE
Victron Multiplus 2 5000 - JK BMS only charging with 35 - 37.2 A

### DISCUSSION THREAD

**Discussion by leinadsims** [Louisvdw/dbus-serialbattery#438] [Q&A] [answered]
The Victron is not charging the battery with more then 37.2A. It is a 120Ah battery and the original BMS was able to charge and discharge with 100A (had other big disadvantages). Since I changed the BMS to JK BD6A20S10P, which also allows 100A contnious charage and discharge, and installed this dbus serial battery stuff to my Cerbo, I am not able to bring the environment to higher Amps for load. I checked already all the configuration stuff but from what I see there is only a limitation when getting near SOC oder empty battery. Right now I have around 50% SOC, full sunshine and 1.9kW going into the battery and 1.3kW into grid. 
Did I miss anything. Anybody who has an Idea what to change?

**Comment by leinadsims** [comment]
![image](https://user-images.githubusercontent.com/121604003/215271841-87973a70-4c8f-479a-b27f-9421862006a1.png)

**Comment by leinadsims** [comment]
![image](https://user-images.githubusercontent.com/121604003/215271951-492cc52f-7047-40c2-88db-be92f845846f.png)

**Comment by vincegod** [comment]
What version of the driver are you running? And what temperature is your battery at?

**Comment by leinadsims** [comment]
I am running the latest version 0.14.3 and the temperature is around 19 - 25 degrees Celsius

**Comment by Louisvdw** [comment]
Can you check which device is selected as the battery monitor in the System Setup.
If the serialbattery is selected, then it should match the CCL/DCL in Parameters.

Also show us what your DVCC settings looks like

**Comment by leinadsims** [comment]
![image](https://user-images.githubusercontent.com/121604003/216526975-0b2a2398-45e5-4db6-a4de-83fb563267a7.png)
![image](https://user-images.githubusercontent.com/121604003/216527039-c3c9ad97-a06e-4c97-b13e-cd2cfc8581d0.png)

**Reply by Louisvdw** [reply]
From the battery side that looks fine. I suspect that it much be something to do with the PV inverter setup/settings.

**Comment by leinadsims** [comment]
best weather today unfortunately I was not able to find the issue so far....
![PXL_20230206_094819182](https://user-images.githubusercontent.com/121604003/217256727-8d6c53c0-a6cd-4f10-ace4-e636b9f8ac83.jpg)
![PXL_20230206_094829872 MP](https://user-images.githubusercontent.com/121604003/217256748-7727f775-e728-4ab0-845d-d4c54a49ba37.jpg)
![PXL_20230206_095305194](https://user-images.githubusercontent.com/121604003/217256765-de82be46-23ba-4774-af8a-33a7c43baaa2.jpg)
![PXL_20230206_095313885](https://user-images.githubusercontent.com/121604003/217256779-d016ed6b-5135-4348-b05c-53c9a03c44d5.jpg)
![PXL_20230206_095323529](https://user-images.githubusercontent.com/121604003/217256792-412f4b20-b97e-4818-b1e2-94286fd849e8.jpg)

**Comment by vincegod** [comment]
What settings for you have in the JK BMS for Max Charge Current or Continued Charge Current?

I've just installed a JK 150A BMS and this was set to 25Amp

**Comment by leinadsims** [comment]
This should bei fine
![Screenshot_20230207-195211](https://user-images.githubusercontent.com/121604003/217343353-02e2ddaa-5bcd-4a64-b23e-9eb9812f56c6.png)

![Screenshot_20230207-195207](https://user-images.githubusercontent.com/121604003/217343393-87c199f3-00e7-4fb3-89ff-9393498a6396.png)

**Comment by leinadsims** [comment]
I found the solution and maybe somebody can explain ....I chnaged the Calibrating Curr (A). to 50 and loading amps incresed directly. So now I am wondering what this point is for. I thought it had something to do with auto calibration which seems to be not....

**Reply by vincegod** [reply]
Your Calibrating Voltage is set to 1.00V this should be the voltage of the pack which is 50v ish. The Calibrating Current should be left at 0.100A unless you know what you are doing.

See the manual here https://github.com/NEEY-electronic/JK/blob/JK-BMS/JKBMS%20INSTRUCTION.pdf

**Reply by mr-manuel** [reply, marked as answer]
On my JKBMS the calibrating voltage is the battery voltage. Here you can insert manually the battery voltage measured with a multimeter. This is a helpful video: [JK-BMS: all settings and menus explained. How to program and set parameters.](https://www.youtube.com/watch?v=xTKiOvJ_ejg)
